### ISSUE TITLE
🧹  update deprecated codemirror dependencies

### ISSUE THREAD

**Issue by SarcevicAntonio** [SvelteLab/SvelteLab#199]
`pnpm i` gives the following output:
```
 WARN  deprecated @codemirror/highlight@0.19.8: As of 0.20.0, this package has been split between @lezer/highlight and @codemirror/language
 WARN  deprecated @codemirror/history@0.19.2: As of 0.20.0, this package has been merged into @codemirror/commands
 WARN  deprecated @codemirror/rangeset@0.19.9: As of 0.20.0, this package has been merged into @codemirror/state
 WARN  deprecated @codemirror/text@0.19.6: As of 0.20.0, this package has been merged into @codemirror/state
---
+ @codemirror/highlight 0.19.8 deprecated
+ @codemirror/history 0.19.2 deprecated
```

**Comment by paoloricciuti** [comment]
Yeah I know I'm just a bit hesitant because if I remember correctly they are included in some package peer dependency...but we can try I guess

**Comment by SarcevicAntonio** [comment]
its not something critical, i just wanted to log it somewhere

**Comment by paoloricciuti** [comment]
Can we close this after #235 @SarcevicAntonio ?
